src/main/java/org/commcare/formplayer/screens/FormplayerQueryScreen.java

```diff
@@ -19,27 +11,4 @@ public FormplayerQueryScreen() {
         super(null, null, null);
     }
 
-    /**
-     * @param skipDefaultPromptValues don't apply the default value expressions for query prompts
-     * @return case search url with search prompt values
-     */
-    public URI getUri(boolean skipDefaultPromptValues) {
```

Comment:
This was originally introduced to avoid double encoding of values: https://github.com/dimagi/formplayer/pull/973#discussion_r648995901

But since we're now separating the creation of the values with the serialisation there's no need to have a separate method.